 <launch>
	<node pkg="pid_color_sensor" name="robotPID" type="robot_pid.py">
		<param name="angle_order" type="double" value="0.0"/>
        	<param name="robot_linear_speed" type="double" value="0.05"/>
        	<param name="robot_angular_speed" type="double" value="0.3"/>

    		<param name="propotionnal_constant" type="double" value="0.002"/>
        	<param name="integration_constant" type="double" value="0.0005"/>
        	<param name="derivation_constant" type="double" value="0.1"/>

        	<param name="error_topic" type="String" value="/Error"/>
		<param name="color_topic" type="String" value="/Color"/>
	</node>
 
 </launch>
